docker/package/tezos_voting_wizard.py

```diff
@@ -22,15 +22,14 @@
     "pass": "Submit a vote not influencing the result but contributing to quorum",
 }
 
-
 # Command line argument parsing
 
 parser.add_argument(
     "--network",
     required=False,
     default="mainnet",
     help="Name of the network to vote on. Is 'mainnet' by default, "
-    "but can take the (part after @) name of any custom instance. "
+    "but can take the (part after @) name of any custom instance or testnet. "
```

Comment:
Minor nitpick, I'd suggest changing this to:
```suggestion
    "but can be a testnet or the (part after @) name of any custom instance. "
```
because it seems a little bit less confusing: in the current wording one could think of looking for the _part after @ of testnets_ which is not there.